jsonrpc/specification.md

```diff
@@ -0,0 +1,49 @@
+# Portal JSON-RPC specification
+
+The `*` is meant to be a placeholder to substitute different Portal sub-networks `history`, `state`, `beacon`, etc
+
+## portal_*RoutingTableInfo
+
+## portal_*AddEnr
+- Adds the Enr to the respective sub-networks routing table
```

Comment:
This is rather an attempt to add to the routing table. It might fail according to the rules of the routing table implementation/configuration (e.g. bucket already full, ip limit reached, etc.).

Reply:
I will make this more clear